api/cpp/Cargo.toml

```
@@ -23,17 +23,19 @@ crate-type = ["lib", "cdylib"]
 # Note, these features need to be kept in sync (along with their defaults) in
 # the C++ crate's CMakeLists.txt
 [features]
-backend-qt = ["i-slint-backend-selector/i-slint-backend-qt"]
 interpreter = ["slint-interpreter"]
```

So we don't need the rtti-features at all

```suggestion
interpreter = ["slint-interpreter/rtti"]
```

---

In fact, rtti should just be enabled by default, so no need even to be a feature.